utils/check-style/check-style

```diff
@@ -177,6 +177,17 @@ for test_case in "${tests_with_query_log[@]}"; do
     } || echo "Queries to system.query_log/system.query_thread_log does not have current_database = currentDatabase() condition in $test_case"
 done
 
+# There shouldn't be large jumps between test numbers (since they should be consecutive)
+max_diff=$(
+    find $ROOT_PATH/tests/queries -iname '*.sql' -or -iname '*.sh' -or -iname '*.py' -or -iname '*.j2' |
```

Comment:
Shouldn't we collect only `0_stateless` or do it separately with `1_stateless`. It can lead to false-negative which is ok, though.